### PR TITLE
completed all functionality

### PR DESCRIPTION
**Pull request by jamesmaddern**

### CODE REVIEW

**Review by domh25 — CHANGES_REQUESTED** [review]
-more comments are needed in this project
-more features need to be added to functionality as explained in brief and maybe letter frequency
-report needs more methods
-location of report is unknown when outputting as a txt file

**Comment by hazzakak** [comment]
- More comments are needed. Particularly in the for loop in the program file.
- 2.6GiB TXT file worked very quickly so big up.
- Through testing, txt files work very well but the program accepts an exe file so could provide a check against whether it's a txt file or not.
- Could do with better named variables (s could be better described etc)
- You have consistent naming conventions tho so yup yup.

File: CMP1903M Assessment 1 Base Code/Input.cs [hunk 1]
```diff
@@ -32,16 +32,37 @@ public List<string> manualTextInput()
...
         //Arguments: string (the file path)
-        //Returns: string
+        //Returns: file contents in a list
         //Gets text input from a .txt file
-        public List<string> fileTextInput(string fileName)
+        public List<string> fileTextInput()
         {
-
+            
+            try
```
Comment (Bailey-Ward) [inline CMP1903M Assessment 1 Base Code/Input.cs:48]: Error handling is implemented which is good, might be worth commenting to point it out as per the brief

File: CMP1903M Assessment 1 Base Code/Program.cs [hunk 2]
```diff
@@ -55,13 +56,15 @@ static void Main()
 
             //Create an 'Analyse' object
             //Pass the text input to the 'analyseText' method
-
-
+            Analyse analyse = new Analyse();
```
Comment (Bailey-Ward) [inline CMP1903M Assessment 1 Base Code/Program.cs]: Good use of objects throughout program

File: CMP1903M Assessment 1 Base Code/Analyse.cs [hunk 3]
```diff
@@ -24,12 +24,38 @@ public List<int> analyseText(string input)
...
+                        }
+                        if (vowels.Contains(Char.ToLower(s[i])))
+                        {
+                            values[1]++;
+                        }
+                        else
+                        {
+                            values[2]++;
+                        }
+                    }
```
Comment (Bailey-Ward) [inline CMP1903M Assessment 1 Base Code/Analyse.cs:72]: Would maybe make sense to have the sentence counter in with the other counters? Could add a letter frequency checker too.

**Pull request merged**